.github/ISSUE_TEMPLATE/REPORT_A_BUG.yml

```
@@ -0,0 +1,86 @@
+name: '🐛 Report a bug'
+description: 'Is something not working as you expect?'
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to file a bug report!
+
+        Before posting, please check that the bug hasn't already been:
+        * fixed in the next release (https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
+        * discussed previously (https://github.com/stylelint/stylelint/search)
+
+        You can help us fix the bug more quickly by:
+        1. Figuring out what needs doing and proposing it.
+        2. Writing the code and submitting a PR once the bug is confirmed.
+  - type: textarea
+    id: reproduce-bug
+    attributes:
+      label: 'What steps are needed to reproduce the bug?'
+    validations:
+      required: true
+  - type: textarea
+    id: stylelint-configuration
+    attributes:
+      label: 'What Stylelint configuration is needed to reproduce the bug?'
+      description: 'Provide the smallest reproducible example.'
+      placeholder: |
+        e.g.
+
+        ```json
+        {
+          "rules": {
+            "number-leading-zero": "always"
+          }
+        }
+        ```
+    validations:
+      required: false
+  - type: input
+    id: stylelint-run
+    attributes:
+      label: 'How do you run Stylelint?'
+      description: 'For example, CLI, Node,js API or PostCSS plugin.'
+      placeholder: |
+        e.g. "CLI with `stylelint "**/*.css" --config myconfig.json`"
```

```suggestion
        CLI with `stylelint "**/*.css" --config myconfig.json`
```

Let's remove the "e.g." and quotes as they're redundant in placeholders(, but they aren't before). It'll mean more of the placeholder content it show too.